var/spack/repos/builtin/packages/py-virtualenv/package.py

```diff
@@ -22,20 +23,28 @@ class PyVirtualenv(PythonPackage):
     version("1.11.6", sha256="3e7a4c151e2ee97f51db0215bfd2a073b04a91e9786df6cb67c916f16abe04f7")
 
     depends_on("python@2.7:2.8,3.4:", type=("build", "run"))
-    depends_on("python@2.7:2.8,3.5:", type=("build", "run"), when="@20.10.0:")
+    depends_on("python@2.7:2.8,3.5:", when="@20.10.0:", type=("build", "run"))
+    depends_on("python@3.6:", when="@20.16.4:", type=("build", "run"))
 
     # not just build-time, requires pkg_resources
     depends_on("py-setuptools@40.6.3:", type=("build", "run"))
-    depends_on("py-setuptools@41.00.03:", type=("build", "run"), when="@20.10.0:")
-    depends_on("py-setuptools-scm@2:", type=("build", "run"), when="@20.10.0:")
+    depends_on("py-setuptools@41.00.03:", when="@20.10.0:", type=("build", "run"))
+    depends_on("py-setuptools-scm@2:", when="@20.10.0:", type=("build", "run"))
```

Comment:
These need to be updated from `pyproject.toml`:
```toml
requires = [
    "setuptools>=59.6",
    "setuptools-scm>=6.4.2",
]
```
Also, setuptools-scm prob is only needed at build time.